static/src/javascripts/bootstraps/commercial.js

```diff
@@ -75,6 +77,9 @@ define([
         ['cm-ready', function () {
             mediator.emit('page:commercial:ready');
             userTiming.mark('commercial end');
+            robust.catchErrorsAndLog('ga-user-timing-commercial-end', function () {
```

Comment:
That's pretty nice resolution, I remember you mentioning that, would be interested to see your approach. But yeah, for now, I'll keep it in place and see how we get on (I'm not worried about number of calls to GA itself as there's a net reduction with this PR, but maybe it could make sense to hold back these calls until everything is loaded to avoid clogging the connections - will look at that separately though).

Thanks!